### PR TITLE
docs: added keywords and descriptions for most recent blogs, events, and faqs

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

added keywords and descriptions for most recent blogs, events, and faqs.

### CODE REVIEW

**Comment by yzeng25** [comment]
Hi  @Serendipity96 @bzp2010 @liuxiran @juzhiyuan ,

please take a look when available, thanks!

File: website/events/2021-06-15-release-apache-apisix-dashboard-2.7.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 ---
 title: Release Apache APISIX Dashboard 2.7
+keywords:
+- apisix
```
Comment (juzhiyuan) [inline website/events/2021-06-15-release-apache-apisix-dashboard-2.7.md]: I don't think we need too many duplicated keywords, Search Engine will take them as the same keyword AFAIK 🤔

Comment (yzeng25) [reply website/events/2021-06-15-release-apache-apisix-dashboard-2.7.md]: so we can keep 'apisix' and delete others such as "APISIX"?

Comment (yzeng25) [reply website/events/2021-06-15-release-apache-apisix-dashboard-2.7.md]: deleted duplicated keywords 'apisix' and 'apache apisix'

File: website/events/2021-05-24-release-apache-apisix-2.6.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+- api6
+- API gateway
+- API hosting
+- Security
+- Apache APISIX
+- apisix gateway
+- Ingress Controller
+- Service Mesh
+- Microservices
+- Subscribe Guide
```
Comment (liuxiran) [inline website/events/2021-05-24-release-apache-apisix-2.6.md]: fond nothing about subscribe guide,  better to remove it

File: website/docs/general/security-guide.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+- api6
+- API gateway
+- API hosting
+- Security
+- Apache APISIX
+- apisix gateway
+- Ingress Controller
+- Service Mesh
+- Microservices
+- Release Guide
```
Comment (liuxiran) [inline website/docs/general/security-guide.md:15]: found nothing about release guide

Comment (yzeng25) [reply website/docs/general/security-guide.md:15]: removed

**Review by liuxiran — APPROVED** [review]

File: website/docs/general/security-guide.md (hunk 3 above)
Comment (juzhiyuan) [reply website/docs/general/security-guide.md:15]: Do we still need this?

**Comment by liuxiran** [comment]
cc @moonming for review, thanks

**Review by bzp2010 — APPROVED** [review]

File: website/docs/general/security-guide.md (hunk 3 above)
Comment (yzeng25) [reply website/docs/general/security-guide.md:15]: Yes, I suggest keep these keywords for now. Adjust them based on performance later.

**Pull request merged**